packages/components/src/form-toggle/README.md

```diff
@@ -61,8 +61,8 @@ const MyFormToggle = () => {
 	const [ isChecked, setChecked ] = useState( true );
 
 	<FormToggle
-		checked={ checked }
-		onChange={ () => setChecked( ( state ) => ! state ) }
+		checked={ isChecked }
+		onChange={ () => setIsChecked( ( state ) => ! state ) }
```

Comment:
Thanks for contributing, @imangm. The `setChecked` is the correct function name. The only variable name was incorrect.

Reply:
> 

Thank you! I forgot to update the setter function name in the state. Isn't it better that we change the function name both in the state declaration and in onChange method to setIsChecked?

Reply:
100%. It will match the general `useState` naming convention :)